frontend/src/components/tracks/data/pm.ts

```diff
@@ -0,0 +1,41 @@
+export const or = {
```

Comment:
What does `or` stand for?

Reply:
👍 we should come up with something better here, it's an abbreviation for offer & responsibilities